### ISSUE TITLE
Director / Admin Speech Notes

### ISSUE THREAD

**Issue by Zanger67** [gtmun/couchmun#21]
Could on the admin / stats side of a running timer / speakers list, have a spot where notes on the speech could be made: a text box that auto saves. This could be put on another page where you could "click" a delegate and see their past speeches and notes. Could also instead / in addition have a page with all speeches listed with notes on a side -- sortable by either the delegate if clicked to highlight or sorted by time i.e. recent. Could also have this page as just a standalone since not every speech needs notes all the time tbh...

TL;DR:
- Note taking feature for each speech
- Page with all speeches with notes listed sorted default by recent
  - Click a delegate and it'll filter just theirs

**Comment by Zanger67** [comment]
Similar to / could be tied to development of #22
